var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -173,7 +173,7 @@ class Trilinos(CMakePackage):
     conflicts('+superlu-dist', when='+superlu')
     # For Trilinos v11 we need to force SuperLUDist=OFF, since only the
     # deprecated SuperLUDist v3.3 together with an Amesos patch is working.
-    conflicts('+superlu-dist', when='@:11.14.3')
```

Comment:
please add a `FIXME` above saying that there is a conretization bug. This `conflicts` statement is correct.